packages/scene-composer/tests/components/panels/CustomColorPicker/CustomColorPicker.spec.tsx

```diff
@@ -0,0 +1,188 @@
+import { act, render } from '@testing-library/react';
+import React from 'react';
+import wrapper from '@awsui/components-react/test-utils/dom';
+
+import {
+  CustomColorPicker,
+  DEFAULT_COLOR_PICKER_COLOR,
+} from '../../../../src/components/panels/CustomColorPicker/CustomColorPicker';
+
+jest.mock('@awsui/components-react', () => ({
+  ...jest.requireActual('@awsui/components-react'),
+}));
```

Comment:
Good to know, will fix. I was wondering why this code was used all over the tests since it seems odd (and thought I remembered something from last summer but I wasn't sure...)